code/processes/dqe.q

```diff
@@ -88,10 +88,28 @@ runcheck:{[id;fn;vars;rs]
 
 results:([]id:`long$();funct:`$();vars:`$();procs:`$();procschk:`$();starttime:`timestamp$();endtime:`timestamp$();output:`boolean$();descp:();chkstatus:`$());
 
+/loadtimer:{[DICT]
+/  DICT[`params]:`$";" vs string DICT[`params];                                                                  /- Accounting for potential multiple parameters
+/  .timer.repeat[DICT[`starttime];DICT[`endtime];DICT[`period];(`.dqe.runcheck;DICT[`checkid];.Q.dd[`.dqe;DICT[`action]];DICT[`params];DICT[`procname]);"Running Check on ",string DICT[`proctype]]
+/  }
+
+loadtimer:{[DICT]
+  DICT[`params]:`$";" vs string DICT[`params];                                                                  /- Accounting for potential multiple parameters
+  $[DICT[`mode]=`repeat;
+  .timer.repeat[DICT[`starttime];DICT[`endtime];DICT[`period];(`.dqe.runcheck;DICT[`checkid];.Q.dd[`.dqe;DICT[`action]];DICT[`params];DICT[`procname]);"Running check on ",string DICT[`proctype]];
+  .timer.once[DICT[`starttime];(`.dqe.runcheck;DICT[`checkid];.Q.dd[`.dqe;DICT[`action]];DICT[`params];DICT[`procname]);"Running check once on ",string DICT[`proctype]]]
```

Comment:
Same here, less brackets more readability IMO:
```suggestion
    .timer.once[DICT`starttime;(`.dqe.runcheck;DICT`checkid;.Q.dd[`.dqe;DICT`action];DICT`params;DICT`procname);"Running check once on ",string DICT`proctype]]
```
One other thing to consider is that both these function take very similar args, any way to slim this down?